src/use-deque.ts

```diff
@@ -1,22 +1,22 @@
 import { useState, useMemo } from 'react'
 import produce from 'immer'
 
-export const useDeque = <T>(initialState: T[] = []) => {
```

Comment:
I don't get it... Why changing from generic to fixed type?

Reply:
Drugs! I don't know why I did it